src/SFA.DAS.Payments.RequiredPayments.Tests.Specs/Features/Calculator.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Calculator
```

Comment:
Is this needed? Looks like a hello world type example

Reply:
Removed